resources/assets/sass/app.scss

```diff
@@ -225,7 +225,7 @@ h1 {
       background-color: lighten($arc_rose, 15%);
       color: $arc_white;
     }
-    &#updateMarket {
+    &.updateTrader {
```

Comment:
Does this affect the market forms which might use updateMarket on thier buttons?

https://github.com/neontribe/ARCVService/blob/1.12/release/resources/views/service/markets/create.blade.php
https://github.com/neontribe/ARCVService/blob/1.12/release/resources/views/service/markets/edit.blade.php

Reply:
no, those are nicely spaced.

Reply:
the original css property was not intended to target those in the makert create and edit pages. it was just change of the selector that lead to that. but there's no need, those buttons are nicely spaced, not overlapping with any other elements event at smaller screen sizes.